stripcaged.lua

```diff
@@ -1,8 +1,29 @@
+local rawargs = {...}
+local args = require 'utils'.invert(rawargs)
```

Comment:
you are better off using `argparse.processArgsGetopt()`. it returns the non-option params as a list.

Reply:
Thanks. My latest commits try to address this.